docs/coordinates/representations.rst

```diff
@@ -32,21 +32,32 @@ The built-in representation classes are:
   (``rho``), azimuthal angle (``phi``), and height (``z``).
 
 
-Astropy also offers a `~astropy.coordinates.BaseGeodeticRepresentation` useful to
+Astropy also offers a `~astropy.coordinates.BaseGeodeticRepresentation` and
+a `~astropy.coordinates.BaseBodycentricRepresentation` useful to
 create specific representations on spheroidal bodies.
-This is used internally for the standard Earth ellipsoids used in
+`~astropy.coordinates.BaseGeodeticRepresentation` is used internally for the standard
+Earth ellipsoids used in
 `~astropy.coordinates.EarthLocation`
```

Comment:
Maybe combine with last line while you are making the last changes.

Reply:
Sorry I'm not sure to understand this comment....

Reply:
Meant to combine the two lines to give```
Earth ellipsoids used in `~astropy.coordinates.EarthLocation`
```